themes/ddbasic/scripts/ddbasic.common.js

```diff
@@ -4,13 +4,6 @@
    * Toggle opening hours
    */
   function toggle_opening_hours() {
-    // Create toggle link
-    $('<a />', {
-      'class' : 'opening-hours-toggle js-opening-hours-toggle js-collapsed',
-      'href' : Drupal.t('#toggle-opening-hours'),
-      'text' : Drupal.t('Opening hours')
-    }).insertBefore('.js-opening-hours-toggle-element');
-
```

Comment:
As I see it this moves rendering of the toggle link from JS to PHP (template.php line 283). What is the reason behind this change?